frontend/src/components/Board/Board.jsx

```diff
@@ -5,36 +5,31 @@ import {
   BoardTitle,
   AuthorInfoContainer,
   AuthorName,
-  AuthorAvatar
-} from "./Board.styled";
-import DotsButton from "../DotsButton/DotsButton";
+  AuthorAvatar,
+} from './Board.styled';
+import DotsButton from '../DotsButton/DotsButton';
 import Dropdown from '../Dropdown/Dropdown';
 import { FaUserCircle } from 'react-icons/fa';
 
-const Board = ({user, boardTitle}) => {
+const Board = ({ user, boardTitle }) => {
   return (
-      <BoardInfoContainer>
-        <BoardInfo>
-          <BoardImage />
-          {/* TODO: Dodać Link dookoła BoardTitle jak już będzie widok Board'a  */}
-          <BoardTitle>
-            {boardTitle}
-          </BoardTitle>
-          <DotsButton>
-            <Dropdown>
-            </Dropdown>
-          </DotsButton>
-        </BoardInfo>
-        <AuthorInfoContainer>
-          <AuthorName>
-            {user}
-          </AuthorName>
-          <AuthorAvatar >
-            <FaUserCircle />
-          </AuthorAvatar>
-        </AuthorInfoContainer>
-      </BoardInfoContainer>
+    <BoardInfoContainer>
+      <BoardInfo>
+        <BoardImage />
+        {/* TODO: Dodać Link dookoła BoardTitle jak już będzie widok Board'a  */}
+        <BoardTitle>{boardTitle}</BoardTitle>
+        <DotsButton>
+          <Dropdown></Dropdown>
+        </DotsButton>
+      </BoardInfo>
+      <AuthorInfoContainer>
+        <AuthorName>{user}</AuthorName>
+        <AuthorAvatar>
+          <FaUserCircle />
+        </AuthorAvatar>
+      </AuthorInfoContainer>
+    </BoardInfoContainer>
   );
-}
+};
 
-export default Board;
\ No newline at end of file
+export default Board;
```

Comment:
jak te zmiany się imają do recovery password?

Reply:
to raczej błąd IDE

Reply:
a już wiem co to, prettier pozmieniał ułożenie :)